src/Aspire.Hosting/Eventing/DistributedApplicationEventSubscription.cs

```diff
@@ -0,0 +1,30 @@
+using System.Diagnostics.CodeAnalysis;
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Eventing;
+
+/// <summary>
+/// Represents a subscription to an event that is published during the lifecycle of the AppHost.
+/// </summary>
+/// <param name="callback">Callback to invoke when the event is published.</param>
+[Experimental("ASPIREEVENTING001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
```

Comment:
Do we really need it to be `ASPIREEVENTING001`? How many "eventing" warnings do we image there will be?

Instead, should it just be `ASPIRE00x`?

Reply:
I like them to be a bit more verbose because it means you don't need to lookup what it means :)